pkg/cmd/pr/checkout/checkout.go

```diff
@@ -65,6 +67,7 @@ func NewCmdCheckout(f *cmdutil.Factory, runF func(*CheckoutOptions) error) *cobr
 	cmd.Flags().BoolVarP(&opts.RecurseSubmodules, "recurse-submodules", "", false, "Update all submodules after checkout")
 	cmd.Flags().BoolVarP(&opts.Force, "force", "f", false, "Reset the existing local branch to the latest state of the pull request")
 	cmd.Flags().BoolVarP(&opts.Detach, "detach", "", false, "Checkout PR with a detached HEAD")
+	cmd.Flags().StringVarP(&opts.Local, "branch", "b", "", "Rename branch during checkout")
```

Comment:
Let's be a bit more specific about which branch exactly gets renamed, since we're checking out a remote branch here, and no remote branches will be affected. How does something like `Local branch name to use (default: the name of the head branch)` sound?